.github/workflows/es-spellcheck.yml

```
@@ -11,19 +11,30 @@
 name: Spellcheck CI
 # yamllint disable-line rule:truthy
 on:
-  pull_request:
-    paths:
-      - 'content/es/**/*.md'
+  push:
     branches:
-      - dev-es
+      - 'test-ramrodo-action'
+  # pull_request:
+    # paths:
+    #   - 'content/es/**/*.md'
+    # branches:
+    #   - *
   workflow_dispatch:
 
 jobs:
   check-spanish-spellcheck:
     name: Run PySpelling tool to verify spanish spelling issues
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+      - name: Check word list sorts
+        run: |
+          wordlist="content/es/.wordlist.txt"
+          wordsUnsorted=$(diff --changed-group-format="%>%<" --unchanged-group-format="" <(cat $wordlist) <(LC_ALL= sort -f $wordlist))
+          if [[ -n "$wordsUnsorted" ]]; then
+            echo "$wordsUnsorted"
+            exit 1
+          fi
```

Me parece que con solo propagar el error es más que suficiente
```suggestion
          set -o errexit
          diff content/es/.wordlist.txt <(LC_ALL= sort -f content/es/.wordlist.txt) ```